testCourse/courseInstances/Sp15/assessments/hw2-miscProblems/infoAssessment.json

```diff
@@ -37,7 +37,8 @@
                 {"id": "fossilFuelsRadio",           "points": 2, "maxPoints": 10},
                 {"id": "positionTimeGraph",          "points": 2, "maxPoints": 10},
                 {"id": "downloadFile",               "points": 2, "maxPoints": 10},
-                {"id": "differentiatePolynomial",    "points": 2, "maxPoints": 10}
+                {"id": "differentiatePolynomial",    "points": 2, "maxPoints": 10},
+                {"id": "subfolder/nestedQuestion",   "points": 2, "maxPoints": 10}
```

Comment:
Any chance this could be expanded with a second nested question, e.g. `subfolder/question2`?

Reply:
I can, any specific reason for it though?

Reply:
Probably adds more robustness for an actual application inside of the test suite.